package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "dao-v2-ui",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
Keep attention here. You are probably using a newer version of node than the one used in the project. Try to check if there's a nvmrc and stick to that version. I wouldn't modify the lock file in this PR.